client/src/pages/auth/Sign-up.js

```diff
@@ -28,26 +29,24 @@ const SignupSchema = Yup.object().shape({
 
 const Signup = () => {
     const classes = useStyles();
-    const [school, setSchool] = useState('');
     const [course, setCourse] = useState('');
-    const [schoolCourses, setSchoolCourses] = useState(courses);
-    // const [addedCourses, setAddedCourses] = useState([]);
+    const [schoolCourses] = useState(courses);
+    const [addedCourses, setAddedCourses] = useState([]);
+    const [selectedId, setSelectedId] = useState([]);
```

Comment:
Any reason for storing the ids separately here?

Reply:
Yes, it is a duplicate state. It will be fixed in the refactored merge.